gui/options.cpp

```diff
@@ -2098,6 +2109,14 @@ void GlobalOptionsDialog::addPathsControls(GuiObject *boss, const Common::String
 
 	_themePathClearButton = addClearButton(boss, prefix + "ThemePathClearButton", kThemePathClearCmd);
 
+	if (!lowres)
```

Comment:
Braced if-else's are preferred. (Some cases below as well)

Reply:
The older code for theme, saves and extra path still has un-braced if-else, shall I update those as well?